.gitpod.yml

```diff
@@ -0,0 +1,27 @@
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
+tasks:
+  - name: Install npm depdencies
+    init: |
+      npm i -g npm
+      npm install
+    command: |
+      npm i -g npm
+      sleep 30
```

Comment:
Added sleep here
`gp await-port` doesnt seem to work with stargate, as stargate opens the ports before the actual image is initialized successfull. (At least I have the impression)

Reply:
oh ok. If we need the first version to have a sleep I think that is ok, we can have an issue/todo for it, where we can improve it for the next version.